### PR TITLE
fix(suggestion-bucket):removed email

### PR DESCRIPTION
**Pull request by ponrajaprabhusk**
### Functionality:
 Fixed the suggestion-bucket

### Solution:
Removed the email tag in suggestion bucket

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
<-- please tell us how to test functionality/bug -->

### CODE REVIEW

File: src/app/body/common-tools/suggestion-bucket/suggestion-bucket.component.html [hunk 1]
```diff
@@ -22,9 +22,7 @@
                     {{userName}}
                 </span>
                 <br>
-                <span id="email">
-                    {{email}}
-                </span>
```
Comment (viv9k) [inline src/app/body/common-tools/suggestion-bucket/suggestion-bucket.component.html:27]: Please don't remove emails from here rather receive an input from parent components if we want to show email or not

**Review by viv9k — CHANGES_REQUESTED** [review]
Need changes

File: src/app/body/common-tools/suggestion-bucket/suggestion-bucket.component.html (hunk 1 above)
Comment (ponrajaprabhusk) [reply src/app/body/common-tools/suggestion-bucket/suggestion-bucket.component.html:27]: Done !!

File: src/app/body/common-tools/suggestion-bucket/suggestion-bucket.component.ts [hunk 2]
```diff
@@ -26,6 +26,8 @@ export class SuggestionBucketComponent implements OnInit {
 
   @Input("email") email: string;
   @Input("showOnlyProfilePic") showOnlyProfilePic: boolean = false;
+
+  @Input("showEmail") showEmail: boolean = false;
```
Comment (viv9k) [inline src/app/body/common-tools/suggestion-bucket/suggestion-bucket.component.ts:30]: You will have to receive this value from all the parents calling this component

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required fix

File: src/app/body/common-tools/suggestion-bucket/suggestion-bucket.component.ts (hunk 2 above)
Comment (ponrajaprabhusk) [reply src/app/body/common-tools/suggestion-bucket/suggestion-bucket.component.ts]: Done

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**